build-system/tasks/extension-generator/jscodeshift/insert-extension-bundles-config.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
```

Comment:
I don't know anything about jscodeshift so someone else should definitely review this :)